sympy/vector/functions.py

```diff
@@ -474,3 +474,54 @@ def _path(from_object, to_object):
         from_path.append(other_path[i])
         i -= 1
     return index, from_path
+
+def orthogonalize(vlist, orthonormal=False):
+    """
+    Takes a set of independent vectors and orthogonalizes them
+    using the Gram - Schmidt process. Returns a set of
+    orthogonal or orthonormal vectors.
+
+    Parameters
+    ==========
+
+    vlist : list of independent vectors to be made orthogonal.
```

Comment:
Please change the comment here to "iterable of independent..." or "sequence of independent...".